roles/pulp_webserver/tasks/main.yml

```diff
@@ -80,6 +80,14 @@
   loop_control:
     loop_var: __pulp_webserver_acme_path
 
+- name: Include galaxy_ng snippet dependencies
+  set_fact:
+    pulp_install_plugins_normalized: "{{ pulp_install_plugins_normalized|combine({'pulp-ansible': {}, 'pulp-container': {}}) }}"
+  when:
+    - pulp_install_plugins_normalized['galaxy-ng'] is defined
+    - pulp_install_plugins_normalized['pulp-ansible'] is undefined
+    - pulp_install_plugins_normalized['pulp-container'] is undefined
```

Comment:
Nice, but what happens if galaxy-ng is defined and pulp-ansible is defined, but pulp-container isn't?

Same for galaxy-ng and pulp-container, but not pulp-ansible.